app/router.ts

```diff
@@ -18,8 +19,13 @@ const Router = EmberRouter.extend({
         scheduleOnce('afterRender', this, () => {
             const page = this.get('url');
             const title = this.getWithDefault('currentRouteName', 'unknown');
+            const authenticated = this.get('session.isAuthenticated') ? 'Logged in' : 'Logged out';
```

Comment:
Probably should be 'true' or 'false' to be consistent

Reply:
I agree for internal consistency, but unfortunately this feature is already implemented by the Platform team, so I'm matching that. https://openscience.atlassian.net/browse/PLAT-306

Reply:
That is unfortunate, but now would be the best time to correct it in both places.